src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/CreateReportRequest.kt

```diff
@@ -32,10 +32,16 @@ data class CreateReportRequest(
   val reportedDate: LocalDateTime,
   // TODO: there is not yet a way to add any more details to a report, question-response pairs, etc
 ) {
-  fun toNewEntity(incidentNumber: String, event: Event, createdBy: String, clock: Clock): Report {
+  fun validate() {
```

Comment:
this should probably be private

Reply:
hang on, I need to have a look at this better as I thought this was invoked directly from within `toNewEntity()` 🤔